blocks/api/test/parser.js

```diff
@@ -99,15 +106,21 @@ describe( 'block parser', () => {
 		} );
 
 		it( 'should create the requested block with no attributes if it exists', () => {
-			registerBlock( 'core/test-block', {} );
+			registerBlock( 'core/test-block', {
+				edit: edit,
```

Comment:
Yeah, I wasn't sure what style should be done for that.  In some projects people shy away from that as it can be confusing as to how that works.